client/layout/guided-tours/config-elements/continue.js

```diff
@@ -90,10 +90,12 @@ export default class Continue extends Component {
 			return null;
 		}
 
+		const instructionClassName = 'guided-tours__actionstep-instructions';
```

Comment:
Our linter didn't like the class name but I thought it was appropriate so I moved it into a variable.